sanic/app.py

```diff
@@ -967,34 +969,47 @@ def test_client(self):
 
     def run(
         self,
-        host=None,
-        port=None,
-        debug=False,
-        ssl=None,
-        sock=None,
-        workers=1,
-        protocol=None,
-        backlog=100,
-        stop_event=None,
-        register_sys_signals=True,
-        access_log=True,
-        **kwargs
-    ):
+        host: typing.Optional[str] = None,
+        port: typing.Optional[int] = None,
+        debug: bool = False,
+        ssl: typing.Union[dict, SSLContext, None] = None,
+        sock: typing.Optional[socket] = None,
+        workers: int = 1,
+        protocol: typing.Type[Protocol] = None,
+        backlog: int = 100,
+        stop_event: typing.Any = None,
+        register_sys_signals: bool = True,
+        access_log: typing.Optional[bool] = None,
+        **kwargs: typing.Any
```

Comment:
Thanks for adding typings, can we refactor a little bit ? like `from typing import Any, Union, ...` and use them directly here.